rosette/lib/trace/raco.rkt

```diff
@@ -0,0 +1,134 @@
+#lang racket/base
+
+(require racket/cmdline
+         racket/format
+         racket/match
+         racket/string
+         racket/list
+         racket/exn
+         syntax/to-string
+         raco/command-name
+         "../util/module.rkt"
+         "compile.rkt"
+         "tool.rkt")
+
+(define symbolic-trace-streaming? (make-parameter #f))
+(define symbolic-trace-show-stats? (make-parameter #f))
+(define symbolic-trace-context-length (make-parameter 3))
+
+(define module-name (make-parameter 'main))
+(define file
+  (command-line
+   #:program (short-program+command-name)
+   ; Tool configuration
+   #:help-labels "" "Tracing settings"
+   #:once-each
+
+   ;; SymPro options
```

Comment:
stale comment

Reply:
That's actually intentional. I meant to share flags that are similar to SymPro as much as possible.